src/algokit/__init__.py

```diff
@@ -1,16 +1,6 @@
-import platform
 import sys
 
 # this isn't beautiful, but to avoid confusing user errors we need this check before we start importing our own modules
-if sys.version_info < (3, 10, 0):
```

Comment:
Why is this removed?

Reply:
`Version block is outdated for minimum Python versionRuff[UP036](https://beta.ruff.rs/docs/rules/outdated-version-block)` ruff behaviour in my IDE, i guess my global ruff version is higher than the one in venv. Will revert that change manually